discovery-provider/src/tasks/index.py

```diff
@@ -751,6 +757,27 @@ def revert_blocks(self, db, revert_blocks_list):
                 logger.info(f"Reverting associated Wallet: {user_id}")
                 session.delete(associated_wallets_to_revert)
 
+            for user_events_to_revert in revert_user_events:
```

Comment:
Nit: could maybe pull this out into a `revert_user_events` fn so we're not adding more entropy to this rather long guy